api/envoy/config/core/v3/protocol.proto

```diff
@@ -244,10 +244,7 @@ message Http1ProtocolOptions {
 
 message KeepaliveSettings {
   // Send HTTP/2 PING frames at this period, in order to test that the connection is still alive.
-  google.protobuf.Duration interval = 1 [(validate.rules).duration = {
-    required: true
-    gte {nanos: 1000000}
-  }];
+  google.protobuf.Duration interval = 1 [(validate.rules).duration = {gte {nanos: 1000000}}];
```

Comment:
Is this 1 second? That seems a bit on the short side to me, as Chrome uses 10 seconds, I think. Admittedly this code isn't changing as part of this PR, but perhaps something to think about.

If this is not present then presumably no keep-alives are sent? Worth mentioning?

Reply:
Wow! So like 1ms? This is *really* low. That's well below RTT, right? Does this mean we'll send a ping every 1ms?

Reply:
this is just validation rules - it means if you set an interval lower than 1ms it'll reject the config. it's not a suggested (or advised!) value :-)

Reply:
OH! *whew* OK, that makes a lot more sense :) Thanks!